default/templates/default/snippets/optional_arguments_scripts.html

```diff
@@ -0,0 +1,12 @@
+<script type="text/javascript">
```

Comment:
Please do not define snippets to include Javascript code. Create a .js file instead.

Reply:
We must change `published_date_scripts.html` too? I think that's where Rodrigo's idea came from.

Reply:
If possible, yes. I understand this is Rodrigo's code, but I think we should correct or improve what we can.

Reply:
Created `optional_arguments.js `and `published_date.js`.

Reply:
Also created `url_uploader.js`